src/main/java/graphql/schema/TypeTraverser.java

```diff
@@ -0,0 +1,62 @@
+package graphql.schema;
+
+
+import graphql.util.TraversalControl;
+import graphql.util.Traverser;
+import graphql.util.TraverserContext;
+import graphql.util.TraverserResult;
+import graphql.util.TraverserVisitor;
+
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+
+public class TypeTraverser {
```

Comment:
Java Doc and @Internal of @PublicApi  would be needed

Reply:
TypeTraverser is similar to [NodeTraverser](https://github.com/graphql-java/graphql-java/blob/master/src/main/java/graphql/language/NodeTraverser.java). Shall I update both?

I consider NodeTraverser/TypeTraverser as @Internal for now.

Reply:
yes, please make it `@Internal` for now.